src/components/PermissionCard.vue

```diff
@@ -70,6 +77,9 @@ export default defineComponent({
                     <div v-for="a in permissionLocal.required_auth.accounts" :key="`${a.permission.actor}-${a.permission.permission}`">
                         <div><span>{{`+${a.weight} &nbsp; &nbsp; `}}</span><span class="text-bold" v-html="formatAccount(a.permission.actor?.toString(), 'account')"></span><span> @{{a.permission.permission}}</span></div>
                     </div>
+                    <div v-for="w in permissionLocal.required_auth.waits" :key="`${w.wait_sec}-${w.weight}`">
+                        <div><span>{{`+${w.weight} &nbsp; &nbsp; `}}</span><span v-html="formatWait(w.wait_sec.value)"></span></div>
```

Comment:
Does this work? it is better to not use `v-html`
https://chat.openai.com/share/863a44da-3728-4fe5-83b8-285f4912abeb
```suggestion
                        <div><span>{{`+${w.weight} &nbsp; &nbsp; `}}</span><span v-html="formatWait(w.wait_sec.value)"></span></div>
```

Reply:
I didn't know. Thanks!